source/linear-algebra/source/05-GT/05.ptx

```diff
@@ -0,0 +1,233 @@
+                The map <m>C_{\mathcal{B}}</m> is a linear map. 
+            </p>
+            <p>
+                Compute <m>C_{\mathcal{B}}\left(\begin{bmatrix}1\\1\\1\end{bmatrix}\right).</m>
+            </p>
+    </task>
+    <task>
+            <p>
+                Compute <m>C_\mathcal{B}(\vec{e}_1),C_\mathcal{B}(\vec{e}_2), C_\mathcal{B}(\vec{e}_3)</m> and, in doing so, write down the standard matrix <m>M_\mathcal{B}</m> of <m>C_\mathcal{B}</m>.
+            </p>
+            <sage language="octave">
+                <input>
+                    
+                </input>
+                <output>
+                    
+                </output>
+            </sage>    
+    </task>
+    
+</activity>
+
+<observation>
+    <p>
+        Note that one way to compute <m>M_{\mathcal{B}}</m> is calculate the RREF of the following matrix:
+        <me>\left[\begin{array}{ccc|ccc}1&amp;1&amp;0&amp;1&amp;0&amp;0\\
+            0&amp;-1&amp;1&amp;0&amp;1&amp;0\\
+            1&amp;1&amp;1&amp;0&amp;0&amp;1\end{array}\right]\sim\left[\begin{array}{ccc|ccc}1&amp;0&amp;0&amp;2&amp;1&amp;-1\\
+            0&amp;1&amp;0&amp;-1&amp;-1&amp;1\\
+            0&amp;0&amp;1&amp;-1&amp;0&amp;1\end{array}\right]</me>
+            Thus, the matrix <m>M_{\mathcal{B}}</m> is the inverse of the matrix <m>[\vec{v}_1\ \vec{v}_2\ \vec{v}_3]</m>.
+            That is:
+            <me>M_{\mathcal{B}}^{-1}=[\vec{v}_1\ \vec{v}_2\ \vec{v}_3].</me>
+    </p>
+</observation>
+<definition xml:id="def-change-of-basis">
+    <statement>
+        <p>
+            Given a basis <m>\cal{B}=\setList{\vec{v}_1,\dots, \vec{v}_n}</m> of <m>\IR^n</m>, the <term>change of basis/coordinate</term> transformation <em>from</em> the standard basis <em>to</em> <m>\mathcal{B}</m> is the transformation <m>C_\mathcal{B}\colon\IR^n\to\IR^n</m> defined by the property that, for any vector <m>\vec{v}\in\IR^n</m>, the vector <m>C_\mathcal{B}(\vec{v})</m> is the unique solution to the vector equation:
+            <me>x_1\vec{v}_1+\dots+x_n\vec{v}_n=\vec{v}.</me>
+            Its standard matrix is called the change-of-basis matrix from the standard basis to <m>\mathcal{B}</m> and is denoted by <m>M_{\mathcal{B}}</m>.
+            It satisfies the following:
+            <me>M_{\mathcal{B}}^{-1}=[\vec{v}_1\ \cdots\ \vec{v}_n].</me>
+        </p>
+        <p>
+            The vector <m>C_\mathcal{B}(\vec{v})</m> is the <m>\mathcal{B}</m>-coordinates of <m>\vec{v}</m>.
+            If you work with standard coordinates, and I work with <m>\mathcal{B}</m>-coordinates, then to build the vector that you call <m>\vec{v}=\begin{bmatrix}a_1\\\vdots\\a_n\end{bmatrix}=a_1\vec{e}_1+\cdots+a_n\vec{e}_n</m>, I would first compute <m>C_\mathcal{B}(\vec{v})=\begin{bmatrix}x_1\\\vdots\\x_n\end{bmatrix}</m> and then build <m>\vec{v}=x_1\vec{v}_1+\cdots+x_n\vec{v}_n</m>.
```

Comment:
Maybe make this part at the end separate from the definition as a remark? Or maybe turn it into an activity?